### ISSUE TITLE
Onboarding - Tab order

### ISSUE THREAD

**Issue by jmgasper** [topcoder-platform/platform-ui#753]
The tab order on the work experience and education popups should be tested.  The work experience tab order seems to skip the "Industry" field
